packages/fhir-router/src/graphql.test.ts

```diff
@@ -976,4 +976,37 @@ describe('GraphQL', () => {
       edges: [{ resource: { name: [{ given: ['Alice'] }] } }],
     });
   });
+
+  test('Connection API without count field', async () => {
```

Comment:
adds coverage for https://sonarcloud.io/code?id=medplum_medplum&pullRequest=1934&selected=medplum_medplum%3Apackages%2Ffhir-router%2Fsrc%2Fgraphql.ts&line=590